bundles/org.eclipse.equinox.security.tests/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Equinox security tests
 Bundle-SymbolicName: org.eclipse.equinox.security.tests;singleton:=true
-Bundle-Version: 1.2.300.qualifier
+Bundle-Version: 1.2.301.qualifier
```

Comment:
Sorry, should have mentioned this before.  The version needs updated in the `pom.xml` also.